apache-rat-core/src/test/java/org/apache/rat/analysis/license/AbstractLicenseTest.java

```diff
@@ -110,8 +107,7 @@ public void testEmbeddedStrings(String id, String family, String name, String no
                 for (String fmt : formats) {
                     boolean found = processText(license, String.format(fmt, target[TEXT]));
                     license.reset();
-                    assertTrue(found, () -> String.format("%s %s did not match pattern '%s' for target string %s", id,
```

Comment:
@Claudenw it seems that assertJ has no equivalent to this supplier construct.
Do you have an idea how to transform these kind of tests into assertJ or should we leave the other assertTrue() - WDYT?

Reply:
I never worked with that construct before and wondered what it is good for. Thx.